### PR TITLE
Adding matcher grey_scrollViewContentOffset to match exact content offset of scroll views

### PR DESCRIPTION
**Pull request by khandpur**

### CODE REVIEW

File: EarlGrey/Matcher/GREYMatchers.m [hunk 1]
```diff
@@ -363,6 +364,22 @@ @implementation GREYMatchers
...
+  MatchesBlock matches = ^BOOL(UIScrollView *element) {
+    return CGPointEqualToPoint([element contentOffset], offset);
+  };
+  DescribeToBlock describe = ^(id<GREYDescription> description) {
+    NSString *desc =
+        [NSString stringWithFormat:@"contentOffset(%@)", NSStringFromCGPoint(offset)];
+    [description appendText:desc];
+  };
+  return grey_allOf(
+      grey_kindOfClass([UIScrollView class]),
```
Comment (tirodkar) [inline EarlGrey/Matcher/GREYMatchers.m]: Is this alignment alright?

File: EarlGrey/Matcher/GREYMatchers.h [hunk 2]
```diff
@@ -14,6 +14,7 @@
 // limitations under the License.
 //
 
+#import <CoreGraphics/CoreGraphics.h>
```
Comment (sid-github) [inline EarlGrey/Matcher/GREYMatchers.h]: Why was this needed? - sorry I missed this earlier.

Comment (khandpur) [inline EarlGrey/Matcher/GREYMatchers.h]: Using CG* ....

File: EarlGrey/Matcher/GREYMatchers.m (hunk 1 above)
Comment (khandpur) [inline EarlGrey/Matcher/GREYMatchers.m]: Fixed.

File: EarlGrey/Matcher/GREYMatchers.h (hunk 2 above)
Comment (sid-github) [inline EarlGrey/Matcher/GREYMatchers.h]: CG* ones are in UIKit  actually but okay it may be a doc error. Resolved.

Comment (khandpur) [inline EarlGrey/Matcher/GREYMatchers.h]: /* CoreGraphics - CGGeometry.h:

/* Points. */

struct CGPoint {
    CGFloat x;
    CGFloat y;
};
typedef struct CGPoint CGPoint;
